lib/cli.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(program) {
      * Trying to load config.
      * Custom config path can be specified using '-c' option.
      */
-    if (!config && !program.preset && !program.autoConfigure) {
+    if (!config && !program.preset && !program.autoConfigure && !program.ignoreMissingConfig) {
```

Comment:
It really depends on how we want this option to behave. What it does currently is true to the option's name: It just ignores the fact that a config is missing, and everything else proceeds as normal.

- This has the advantage that if you always expect an output (especially if you specified a reporter), you will still get an output, as opposed to getting nothing.
- This has the disadvantage of being "a waste of time," and arguably "incorrect." As there are no rules, there is nothing to check, so why claim that we checked anything? (Though [line 81](https://github.com/jacksonrayhamilton/node-jscs/blob/79ef83206549b902f3e2ba25352c0a8675342812/lib/cli.js#L81) doesn't exactly support that argument.)

With `exit(0)`, the behavior would be more consistent with what normally happens when you don't have a config file: The program doesn't do any checking. And with the option enabled, then it wouldn't print an error message either.

In that case, a more correct name would be `--suppress-missing-config-error`. That implies that there is no message, and that there is no erroneous exit code, but it does not imply that we "skip over" missing configs as "ignore" would suggest (rather, we still "stop on them").